draft-ietf-httpbis-compression-dictionary.md

```diff
@@ -264,6 +264,37 @@ longest "match-query" takes precedence.
 1. Given equivalent destination, path and search precedence, the most recently
 fetched dictionary takes precedence.
 
+## Content-Dictionary
+
+When a HTTP server responds with a resource that is encoded with a dictionary
+the server SHOULD send the hash of the dictionary that was used in the
+"Content-Dictionary" response header.
+
+The "Content-Dictionary" response header is a lowercase Base16-encoded
+{{RFC4648}} {{SHA-256}} hash of the contents of the dictionary that was used
+to encode the response.
+
+If the HTTP response contains a "Content-Dictionary" response header with the
+hash of a dictionary that the client does not have available then the client
+MUST NOT attempt to decode or use the HTTP response.
+
+Its syntax is defined by the following {{ABNF}}:
+
+~~~ abnf
+Content-Dictionary = hvalue
+hvalue             = 1*hchar
+hchar              = DIGIT / "a" / "b" / "c" / "d" / "e" / "f"
```

Comment:
This isn't lowercase.  Strings in ABNF are case-insensitive.  You need to use `%x61-66`.

Reply:
Thanks. Updated.